src/views/Pools/index.tsx

```diff
@@ -34,11 +36,12 @@ const Pools: React.FC = () => {
     () => finishedPools.some((pool) => pool.userData && new BigNumber(pool.userData.stakedBalance).isGreaterThan(0)),
     [finishedPools],
   )
+  const cakePool = useMemo(() => openPools.filter((pool) => pool.sousId === 0)[0], [openPools])
```

Comment:
- `autoCakePool` or `cakeVault` would be better name I think, for a sec I thought its manual cake pool
- better to use `openPools.find((pool) => pool.sousId === 0)` instead of `.filter(...)[0]`

Reply:
Yea it _is_ actually the manual pool. I just use the `pool` for some data in the vault. I thought that in theory I could make the vault more easily expandable to other tokens that way. 

Will amend the filter > find, amend the name & add a comment here to clarify.